code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -511,4 +511,32 @@
 	ammo_type = /obj/item/ammo_casing/cap
 	matter = list(DEFAULT_WALL_MATERIAL = 600)
 	max_ammo = 7
-	multiple_sprites = 1
\ No newline at end of file
+	multiple_sprites = 1
+
+
+/obj/item/ammo_magazine/ma12gclip
```

Comment:
That's very...interesting. I thought autolathes already had recycling/exploit prevention in the base code (there's a 1.2x multiplier on production costs, where as recycle costs are exactly what's listed in the code)

I had some issues getting it to compile with the path naming conventions, but i'l take a look at work on it in a bit.